src/test/resources/features/refapp-2.x/03-trackPatient/vitalsAndTriaging.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Vitals Management
+
+  Background:
+    Given a user clicks on Capture Vitals link from Patient dashboard
+    Then the system loads Vitals page
+
+  @selenium
+  @vitals
+  Scenario: Normal Vitals
+    And a user enters normal patient vitals
+    And a user clicks on save button
+    Then the system adds patient vitals into the vitals table 
+   
+  @selenium
+  @vitals 
+  Scenario: Abnormal Vitals
+    And a user enters abnormal patient vitals
+    And a user clicks on save button
```

Comment:
Shouldn't we have a step in here something like `And a user enters normal patient vitals`? The way this scenario reads, it sounds as if you're describing entering invalid vital results and then submitting them and saving them successfully, which is not at all what the actual test is doing.

To make it clearer what this actually verifies, it might be better to write it this way:

```
Scenario: Invalid vitals
  When a user enters a height of "300"
  Then the error message "Maximum: 272" should be displayed
  When a user enters a height of "0"
  Then the error message "MinimumL 10" should be displayed
  When the user enters a height of "170"
  Then the height should be accepted
```

Etc. That way, I can tell from reading the scenario what this actually does.

Reply:
In theory this could be split into multiple scenarios and maybe should be, but I'm not too concerned with that. My concern is that when we see something as failed in the framework, it should be relatively straight-forward to work out *why* it failed without having to dig too deeply into logs.

Reply:
I totally agree with the point @ibacher here especially how the scenario reads: `Scenario: Abnormal vitals`

The ticket description shades out two scenarios under test: `Normal Vitals` and `Abnormal vitals`. For abnormal vitals the test needs to ensure as the user enters a vital out of the range of that specific vital, the system has to shout an alert the end user until  a valid value is entered other the system will not save the vital until a condition is met.

The step `When a user enters abnormal patient vitals ` implementation here all the fields validation within has been catered for. However, I agree the scenario sounds as if the system is saving invalid data which is not true!

I buy the idea of splitting out the second scenario. Probably checking when the user enters a value above the below the minimum vital and as well entering a value over the maximum value.

In the nutshell as I work towards the idea, here is a 2-minutes [video](https://www.youtube.com/watch?v=sbRKwpn3Dzc) for the current test work flow.

Reply:
I love the video... it nicely demonstrates that this all works as expected!

Reply:
Thanks @ibacher  for the video review, I know you have enjoyed it!
I am going to implement the idea have shared in my caption above to rather split the second scenario into two: Entering a value below the minimum vital and entering a value above the maximum vital. Will update the PR over the weekend.